skipruntime-ts/core/src/skipruntime_api.ts

```diff
@@ -276,6 +276,17 @@ export type CollectionUpdate<K extends TJSON, V extends TJSON> = {
   isInitial?: boolean;
 };
 
+/**
+ * A `ReactiveResponse` contains metadata which can be used to initiate and manage reactive
+ * connections to/from a service.  It specifies a `collection` and a current `watermark`,
+ * and is sent by Skip services in the `"Skip-Reactive-Response"` HTTP header by default,
```

Comment:
This does not matter much for now, but I think we should refrain from stating guarantees that the `Skip-Reactive-Response` header will actually contain the collection name + watermark in an intelligible form. For instance, we could have it return a UUID that maps back to something server-side.

Reply:
I don't think this coment states such a guaranee -- even in an unintelligible form, a UUID still "specifies" a collection and watermark.  But agreed that it's not a big deal at the moment, while it's still just direct string-encoded JSON.